test/cloud_testing/platforms/centos8_x86_64_setup.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+
+# source the common platform independent functionality and option parsing
+script_location=$(dirname $(readlink --canonicalize $0))
+. ${script_location}/common_setup.sh
+
+# # Place the overlay directories on ftype=1 16GB xfs partition
+# sudo dd if=/dev/zero of=/xfs-backend bs=$((1024*1024)) count=$((16*1024))
+# sudo mkfs.xfs -n ftype=1 /xfs-backend
+# sudo mkdir -p /var/spool/cvmfs
+# sudo mount /xfs-backend /var/spool/cvmfs
+
+# Place the overlay directories on 16GB ext4 partition
+sudo dd if=/dev/zero of=/ext4-backend bs=$((1024*1024)) count=$((16*1024))
```

Comment:
Is this still necessary, i.e. is the default file system xfs on centos?

Reply:
Yes, XFS is still default on Centos 8.